packages/antd/test/__snapshots__/Array.test.js.snap

```diff
@@ -98,14 +98,16 @@ exports[`array fields array 1`] = `
       </div>
     </fieldset>
   </div>
-  <div>
-    <button
-      className="btn btn-info"
-      type="submit"
-    >
+  <button
```

Comment:
We should remove the span, add the div -- so the snapshots are the same as before.